README.md

```diff
@@ -110,17 +134,15 @@ Endpoint: `Create a UK Property Income & Expenses Period Summary`
 
 * Added TYS downstream error codes for endpoint
 
-
-
-#### Individuals-income-recieved-api
+#### Individuals-income-received-api
 
 Endpoint: `Retrieve an Employment and its Financial Details`
 
-* Added new data field called `offPayrollWorker` to employment
+* Added new data field called `offPayrollWorker` to employment Object
 
 Endpoint: `Create and Amend Employment Financial Details`
 
-* Added new data field called `offPayrollWorker` to Amendemployment
+* Added new data field called `offPayrollWorker` to employment Object
```

Comment:
New field `offPayrollWorker` added to the `employment` object

Reply:
Still not quite right, it was:
New field `offPayrollWorker` added to the `employment` object
(lower-case o in object, and tick-marks around employment)